tasks/windows.ps1

```diff
@@ -0,0 +1,41 @@
+[CmdletBinding()]
+Param(
+  [Parameter(Mandatory = $True)] [String] $master,
+  [Parameter(Mandatory = $False)] [String] $cacert_content,
+  [Parameter(Mandatory = $False)] [String] $certname,
+  [Parameter(Mandatory = $False)] [String] $dns_alt_names
+)
+
+Function Validate-Parameter {
+  Param ($param)
+  if ($param -contains "'") {
+    Throw 'Single-quote is not allowed in arguments'
+  }
+}
+
+Validate-Parameter($certname)
```

Comment:
This should be a validation attribute for the parameters, not separate Function.

Reply:
Ref ref:
https://blogs.technet.microsoft.com/heyscriptingguy/2011/05/15/simplify-your-powershell-script-with-parameter-validation/